tabular/tests/conftest.py

```diff
@@ -103,13 +104,11 @@ def fit_and_validate_dataset(dataset_name, fit_args, sample_size=1000, refit_ful
         directory_prefix = './datasets/'
         train_data, test_data, dataset_info = DatasetLoaderHelper.load_dataset(name=dataset_name, directory_prefix=directory_prefix)
         label_column = dataset_info['label_column']
-        directory = directory_prefix + dataset_name + "/"
-        savedir = directory + 'AutogluonOutput/'
-
-        shutil.rmtree(savedir, ignore_errors=True)  # Delete AutoGluon output directory to ensure runs' information has been removed.
+        savedir = os.path.join(directory_prefix, dataset_name, f'AutogluonOutput_{uuid.uuid4()}')
```

Comment:
Why is this necessary? A conflict could only occur if we were running tests in parallel.

Reply:
This is a good practice. Usually, we'd like to parallelize the pytest testing so we may want to ensure that each individual test is isolated.